### ISSUE TITLE
[Feature Request] shortcut to start the queue

### ISSUE THREAD

**Issue by LePenseur13** [koljapluemer/obsidian-the-queue#4]
More usability. I use the keyboard a lot. 

I assume this would be a valuable but easy addition.

**Comment by koljapluemer** [comment]
You're right, it's already in the backlog, just haven't come around to it. Will let you know.
